CEXT-2387: Update webhooks:dev:run command (#82)
AdobeDocs/commerce-extensibility: src/pages/webhooks/commands.md

```terminal
Module was generated in the app/code/Magento directory
```

## Emulate webhook execution

The `webhooks:dev:run` is used for developing purposes only. It can emulate the execution of your registered webhook with a custom payload.

> This can be helpful during the development or testing of the webhook endpoint as you can emulate webhook execution with different payloads without making required changes in the Adobe Commerce application to trigger webhook execution.

Reviewer: ```suggestion
Run this command after setting the initial webhook payload in a `webhooks.xml` file, Then run the command again any time you make subsequent modifications to the payload until you can confirm that the payload works as expected.
```

### Usage

`bin/magento webhooks:dev:run <webhook-name> <webhook-arguments-payload>`

### Arguments

> &lt;webhook-name> Required. The combination of webhook name and type, the name must begin with either `observer.` or `plugin.`. Example: `observer.checkout_cart_product_add_before:before`

Reviewer: ```suggestion
&lt;webhook-name:type> Required. The combination of webhook name and type. The name must begin with either `observer.` or `plugin.`. The `type` must be either `before` or `after`. Example: `observer.checkout_cart_product_add_before:before`
```

> &lt;webhook-arguments-payload> Required. The webhook arguments payload in JSON format. The payload will be filtered according to the `fields` rules before being sent to the webhook endpoint. This emulates how the real arguments will be filtered in the generated plugin for the webhool.

Reviewer: ```suggestion
&lt;webhook-arguments-payload> Required. The webhook arguments payload in JSON format. The payload will be filtered according to the `fields` rules defined in a `webhooks.xml` file before being sent to the webhook endpoint. This emulates how the real arguments will be filtered in the generated plugin for the webhook.
```

### Example

> For example, you have registered the next webhook:

Reviewer: The `webhooks.xml ` file registered the following webhook:
```

Author: updated

```xml
    <method name="observer.checkout_cart_product_add_before" type="before">
        <hooks>
            <batch>
                <hook name="validate_stock" url="{env:APP_BUILDER_PROJECT_URL}/product-validate-stock" timeout="2000" softTimeout="200" fallbackErrorMessage="The product stock validation failed">
                    <fields>
                        <field name='product.name' source='data.product.name' />
                        <field name='product.sku' source='data.product.sku' />
                    </fields>
                </hook>
            </batch>
        </hooks>
    </method>
```

> Instead of trying to add the product to the cart through the Adobe Commerce UI, you can run the next command with custom payload:

Reviewer: ```suggestion
Instead pf manually adding a product to the cart from the storefront, you can run the following command, which specifies a custom payload:
```

> 

Reviewer: ```suggestion

```bash
```

> `bin/magento webhooks:dev:run observer.checkout_cart_product_add_before:before '{"data":{"product":{"sku":"simple-product","name":"Simple Product"}}}'`

Reviewer: ```suggestion
bin/magento webhooks:dev:run observer.checkout_cart_product_add_before:before '{"data":{"product":{"sku":"simple-product","name":"Simple Product"}}}'
```
```

> The webhook endpoint will receive the next payload according to `fields` configured for the webhook:

Reviewer: ```suggestion
The webhook endpoint receives the following payload, according to `fields` configured for the webhook:
```

```terminal
Module was generated in the app/code/Magento directory
```

## Emulate webhook execution

The `webhooks:dev:run` command is used for development and testing purposes only. It emulates the execution of your registered webhook containing a custom payload with requiring changes to the Commerce application.

> Run this command after setting the initial webhook payload in a `webhooks.xml` file, Then run the command again any time you make subsequent modifications to the payload until you can confirm that the payload works as expected.

Reviewer 2: This is very small and it may be disregarded if not necessary to change, but there's a capital T after a comma: "...payload in a `webhooks.xml` file, Then...". I think there should either be a period to keep the capital T there, or make it a lowercase t. I think second option makes more sense.
